sql/wsrep_thd.cc

```diff
@@ -348,8 +348,13 @@ int wsrep_abort_thd(const THD *bf_thd, THD *victim_thd, bool signal) {
 }
 
 bool wsrep_bf_abort(const THD *bf_thd, THD *victim_thd) {
+  DBUG_TRACE;
   WSREP_LOG_THD(const_cast<THD *>(bf_thd), "BF aborter before");
+
+  //wsrep_lock_thd_query(victim_thd, true);
```

Comment:
Can these commented out lines be removed?

Reply:
Yes, removed.